gotowce/lovelace/wykres-roznych-stanow-jednego-sensora/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Wykres różnych stanów jednego sensora
+nav_exclude: true
+has_children: false
+parent: "Lovelace"
+grand_parent: "Gotowce"
+---
+Lovelace
+{: .label }
+
+# Wykres różnych stanów jednego sensora
+
+## Wygląd wykresu
+
+![Wykres trybu pracy pieca](../../../assets/images/wykres-pracy-pieca-dark.jpg)
+![Wykres trybu pracy pieca](../../../assets/images/wykres-pracy-pieca-light.JPG)
+
+## Opis
+
+Powyższy wykres jest wizualną reprezentacją jednego sensora `sensor.boiler_current_mode`, który - w zależności od aktualnego trybu pracy boilera - przyjmuje stan  `Ogrzewanie`, `Podgrzewanie wody` lub `wyłączony`. Wykres przekształca te stany na wartości `0` (dla stanu `wyłączony`) lub `1` (dla stanów `Ogrzewanie`, `Podgrzewanie wody`) i reprezentuje je w rozbiciu na dwie serie danych.
+
+## Integracje
```

Comment:
To nie jest integracja, tylko customowa karta

Reply:
Masz rację, poprawię